onnx/__init__.py

```diff
@@ -368,3 +368,9 @@ def save_tensor(
 load = load_model
 load_from_string = load_model_from_string
 save = save_model
+
+
+# Override __repr__ for some proto classes to make it more efficient
+ModelProto.__repr__ = object.__repr__  # type: ignore[method-assign]
+GraphProto.__repr__ = object.__repr__  # type: ignore[method-assign]
+FunctionProto.__repr__ = object.__repr__  # type: ignore[method-assign]
```

Comment:
Can we use something like
```py
def model_repr(self):
    return f"Model({self.graph.name})"

def graph_repr(self):
    return f"Graph({self.name})"

onnx.GraphProto.__repr__ = graph_repr
onnx.ModelProto.__repr__ = model_repr
```
Or is there an example to show what `object.__repr__` will produce, in case it is better?

Reply:
Usually model name is empty. Graph name can be empty too for subgraphs. I suppose we need to figure out what is the most informative info to display

Reply:
From the original issue, I think the users does not expect any important info from repr (felels like they just pass by and got stuck). Overwrite it this way might be acceptable.